tox.ini

```diff
@@ -3,8 +3,6 @@ envlist = py26, py27, py32, py33, py34, py35, py36, pypy
 skip-missing-interpreters=True
 
 [testenv]
-deps = nose
-       mock
-       Routes
+deps = .[test_tools,routes_dispatcher]
```

Comment:
This change feels a little bit messy - because it's declaring `cherrypy` (`.`) as a dependency, which I suspect will cause cherrypy to be installed first as a dependency and then again as the project under test. I guess that won't break anything, but it feels messy. Still, I guess it's preferable to have the dependencies declared in just one place.